server/dummyModels/request.js

```diff
@@ -0,0 +1,39 @@
+const requests = [
+  {
+    id: 1,
+    name: 'Okafor-John',
+    request: 'Request to fix the AC',
+    requestDetails: 'I hdjdjjdjfjfjdhhddfhfsbs hbsjhskshdb hfhjsdbjfbhsd hsdjfjbfsdjdshfbdh',
```

Comment:
Why the jibberish? Please real-world examples, not jibberish...imagine if you were presenting this to other non-tech stakeholders

Reply:
Ok. I will do that. Thank you